linera-service/tests/linera_net_tests.rs

```diff
@@ -418,6 +418,123 @@ async fn test_evm_end_to_end_counter(config: impl LineraNetConfig) -> Result<()>
     Ok(())
 }
 
+#[cfg(with_revm)]
+#[cfg_attr(feature = "storage-service", test_case(LocalNetConfig::new_test(Database::Service, Network::Grpc) ; "storage_test_service_grpc"))]
+#[cfg_attr(feature = "scylladb", test_case(LocalNetConfig::new_test(Database::ScyllaDb, Network::Grpc) ; "scylladb_grpc"))]
+#[cfg_attr(feature = "dynamodb", test_case(LocalNetConfig::new_test(Database::DynamoDb, Network::Grpc) ; "aws_grpc"))]
+#[cfg_attr(feature = "kubernetes", test_case(SharedLocalKubernetesNetTestingConfig::new(Network::Grpc, BuildArg::Build) ; "kubernetes_grpc"))]
+#[cfg_attr(feature = "remote-net", test_case(RemoteNetTestingConfig::new(None) ; "remote_net_grpc"))]
+#[test_log::test(tokio::test)]
+async fn test_evm_event(config: impl LineraNetConfig) -> Result<()> {
+    use alloy_primitives::{Bytes, Log, U256};
+    use alloy_sol_types::{sol, SolCall, SolValue};
+    use linera_base::{
+        identifiers::{GenericApplicationId, StreamId, StreamName},
+        vm::EvmQuery,
+    };
+    use linera_execution::test_utils::solidity::get_evm_contract_path;
+    use linera_sdk::abis::evm::EvmAbi;
+    let _guard = INTEGRATION_TEST_GUARD.lock().await;
+    tracing::info!("Starting test {}", test_name!());
+
+    let (mut net, client) = config.instantiate().await?;
+
+    sol! {
+        struct ConstructorArgs {
+            uint64 start_value;
+        }
+        function increment(uint64 input);
+    }
+
+    let start_value = 35;
+    let constructor_argument = ConstructorArgs { start_value };
+    let constructor_argument = constructor_argument.abi_encode();
+
+    let increment = 5;
+
+    let chain = client.load_wallet()?.default_chain().unwrap();
+
+    let (evm_contract, _dir) = get_evm_contract_path("tests/fixtures/evm_example_log.sol")?;
+
+    let instantiation_argument = Vec::new();
+    let application_id = client
+        .publish_and_create::<EvmAbi, Vec<u8>, Vec<u8>>(
+            evm_contract.clone(),
+            evm_contract,
+            VmRuntime::Evm,
+            &constructor_argument,
+            &instantiation_argument,
+            &[],
+            None,
+        )
+        .await?;
+
+    let port = get_node_port().await;
+    let mut node_service = client.run_node_service(port, ProcessInbox::Skip).await?;
+
+    let application = node_service
+        .make_application(&chain, &application_id)
+        .await?;
+
+    let application_id = GenericApplicationId::User(application_id.forget_abi());
+    let stream_name = bcs::to_bytes("ethereum_event")?;
+    let stream_name = StreamName(stream_name);
+
+    let stream_id = StreamId {
+        application_id,
+        stream_name,
+    };
+
+    let mut start_index = 0;
+    let indices_and_events = node_service
+        .events_from_index(&chain, &stream_id, start_index)
+        .await?;
+    let index_and_event = indices_and_events[0].clone();
+    tracing::info!("1: index_and_event={index_and_event:?}");
```

Comment:
(What does the `1:` mean?)

Reply:
Oversight, should have been removed.
For the record, it correspond to the first calls to `events_from_index`.